docs/user-guide/projects/project-lists/README.md

```diff
@@ -148,6 +155,46 @@ By selecting *Overall activity* you can open a view in which all the latest glob
 
 ![overall activity meeting filter](actvity-global-filter.png)
 
+## Favorite project lists
+
+You can mark your private project lists as favorites. This includes shared, public and private project lists, but not the static ones, e.g. *Active projects* or *Archived projects*. To mark a project list as favorite click the star icon in the top right corner. 
+
+![Mark project list as favorite in OpenProject](star-project-list.png)
+
+The star will turn yellow and the favorite project list will move to the top of the list within the respective sidebar section. If multiple project lists are favored, they will be listed alphabetically.
+
+![Favorite project list in OpenProject](star-yellow-project-list.png)
+
+You can remove the star by clicking on the star icon again. 
+
+## Share project lists 
+
+You can share project lists in OpenProject, except the static project lists, e.g. *Active projects* or project lists under *Project status* section in the lefthand menu. Project lists can either be made public, i.e. shared with everyone at OpenProject or with specific users or groups.
+
+### Share with everyone at OpenProject
+
+You can share a project list with everyone at OpenProject, thus making it public. This means that the project list will become visible to all the users registered on your OpenProject instance. 
```

Comment:
@MayaBerd same here, you are not really making it public. And the way users will see this option is always "Share project list with everyone at {instance name}" so maybe we should say "...with everyone in your instance" instead.